dataUtil/constants.py

```diff
@@ -1,2 +1,6 @@
-MS = 1000             # Units of milliseconds
-SEGMENT_LENGTH = 2.5  # Length of calculated non-silent audio segments
+MS = 1000               # Units of milliseconds
+NUM_FREQUENCIES = 70    # Number of frequencies to include in the spectrogram
+NUM_TESTING_CLIPS = 20  # Number of clips to process when testing
+SEGMENT_LENGTH = 5.0    # Length of calculated non-silent audio segments
+SAMPLE_RATE = 22050     # Base sample rate for calculations
+NUM_FFC = 12            # Number of mfccs to extract
```

Comment:
Should be FCC (frequency cepstral coefficients).

also make sure to specify units in the comment for sample rate (Hz), segment length e.g. `# Length of calculated non-silent audio segments (seconds)`